### DISCUSSION TITLE
Performance differences between browser and standalone Enso launch

### DISCUSSION THREAD

**Discussion by xvcgreg** [enso-org/enso#6322] [Help and troubleshooting]
What are the differences in flags between those two launch methods in IDE and Engine?
@farmaazon, @JaroslavTulach, @hubertp

**Comment by wdanilo** [comment]
1. This is a complex topic and the flags depend on many factors.
2. This is rather not an engine-related thing.
3. This question does not have any information why we need to dig now into this topic and I don't think that spending time now on digging there is a time spent well. So closing it for now until it is well described - why we need to talk about it now and what it is about.
